HLTrigger/Configuration/python/HLT_75e33/paths/HLT_IsoMu20_eta2p1_LooseDeepTauPFTauHPS27_eta2p1_CrossL1_cfi.py

```diff
@@ -0,0 +1,67 @@
+import FWCore.ParameterSet.Config as cms
+
+from ..modules.hltAK4PFJetsForTaus_cfi import *
+from ..modules.hltHpsDoublePFTau40TrackPt1MediumChargedIsolation_cfi import *
+from ..modules.hltHpsSelectedPFTausTrackPt1MediumChargedIsolation_cfi import *
+from ..sequences.hgcalLocalRecoSequence_cfi import *
```

Comment:
This looks outdated, see https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalLocalRecoSequence_cfi.py